docs/development-guidelines/UIStylingGuidelines.md

```diff
@@ -428,6 +428,26 @@ Developers should avoid using numeric values directly and instead strive to use
 
 !!! tip "In particular a `OpacityDisabledElement` alias is defined, which should be used for disabled elements."
 
+### Other Resource Palettes
+!!! info "Like Opacity and Colors, the App has palettes for other resources"
+These provide an abstraction over the actual values, and should be used in the Styles of the UI Elements.
```

Comment:
I think the `"` has to be after `UI Elements.`

Reply:
I meant for it to be outside the Info. Updated it